app/api/resources/users.py

```diff
@@ -47,13 +48,18 @@ def post(cls):
 
         data = request.json
 
+        is_valid = validate_user_registration_request_data(data)
+
+        if is_valid != {}:
```

Comment:
if not is_valid:
This is not a good way to do this

Reply:
sorry, missed it. changed it now.